src/output/flat.rs

```diff
@@ -98,10 +98,8 @@ fn is_show_format(name: &str) -> Option<&'static str> {
         "tar.header_with_data" => Some("Tar File Entry"),
 
         // Text
-        "text" => Some("ASCII text"),
-
-        // UTF-8
-        "utf8.string" => Some("UTF-8"),
+        "text.string.ascii" => Some("ASCII String"),
+        "text.string.utf8" => Some("UTF-8 String"),
```

Comment:
Can probably say "ASCII Text" and "UTF-8 Text" I think, given that these are a little more friendly in this context.